package.json

```diff
@@ -9,16 +9,18 @@
   },
   "dependencies": {
     "babel-eslint": "6.0.4",
-    "eslint": "2.4.0",
-    "eslint-config-airbnb": "^6.0.2",
-    "eslint-plugin-babel": "2.1.1",
-    "eslint-plugin-react": "^4.0.0",
-    "glob": "5.0.14",
-    "meld": "^1.3.2"
+    "eslint": "2.10.2",
+    "eslint-config-airbnb": "9.0.1",
+    "eslint-plugin-babel": "3.2.0",
+    "eslint-plugin-import": "1.8.1",
+    "eslint-plugin-jsx-a11y": "1.2.2",
+    "eslint-plugin-react": "5.1.1",
+    "glob": "7.0.3",
+    "meld": "1.3.2"
   },
   "devDependencies": {
-    "chai": "^2.1.0",
-    "mocha": "^2.3.4",
+    "chai": "^3.5.0",
```

Comment:
Cool - we've seen issues elsewhere with different versions of these being installed, but happy to address that whenever it comes up